src/layouts/Video.js

```diff
@@ -62,6 +62,9 @@ const vimeoEmbed = videoId => (
 
 const Video = ({ video }) => {
   const { vimeoId, caption } = video
+  if (!vimeoId) {
+    return null
+  }
```

Comment:
lol nice

Reply:
as an aside, how do you feel about an `autoplay` option in the CMS?

Reply:
lol yep